HinojosaFernando/Biblioteca/src/Libreria.java

```diff
@@ -0,0 +1,9 @@
+public class Libreria {
+    private int cantBooks;
+
```

Comment:
For time being it's okay to have only cantBooks as an attribute but eventually you will have a collection of Books.